src/App.js

```diff
@@ -2,14 +2,29 @@ import React from "react";
 
 import Bookings from "./Bookings";
 import "./App.css";
+import Heading from "./Heading";
+import TouristInfoCard from "./TouristInfoCards";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
+
 
 const App = () => {
+  const contactDetails = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789",
+  ];
```

Comment:
It is great we have put this into a variable, however it could be better to define this outside of the component itself. Right now we are creating a new variable every time it renders, but it never changes! 😄